ui/components/ui/button/buttons.scss

```diff
@@ -20,7 +20,7 @@
   &--disabled,
   &[disabled] {
     cursor: auto;
-    opacity: 0.5;
+    opacity: var(--opacity-disabled);
```

Comment:
Uses the global opacity token which = `0.5`

https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/component-library/component-library-components.scss#L10